src/sql/base/browser/ui/table/plugins/autoSizeColumns.plugin.ts

```diff
@@ -97,6 +97,9 @@ export class AutoColumnSize<T extends Slick.SlickData> implements Slick.Plugin<T
 			}
 
 			let headerWidths: number[] = this.getElementWidths(headerElements);
+			headerWidths = headerWidths.map(width => {
+				return width + 34; // Add to the header width to accommodate for the sort indicator (16px) and the filter menu button (18px)
```

Comment:
plz make this a constant. magic numbers make me feel icky.

Reply:
I'd do this differently, this magic number should be defined in the filter plugin (headerButtonWidth) and passed to this plugin by the grid where these 2 are working together as a parameter extraColumnWidth and add comment. by doing this we are not coupling the plugins.